src/Setup/DevDivInsertionFiles/BuildDevDivInsertionFiles.vb

```diff
@@ -778,6 +800,33 @@ Public Class BuildDevDivInsertionFiles
         xml.Save(GetAbsolutePathInOutputDirectory(PackageName & ".nuspec"), SaveOptions.OmitDuplicateNamespaces)
     End Sub
 
+    Private Sub RewriteVsixManifest(fileToRewrite As String)
+        Dim xml = XDocument.Load(fileToRewrite)
+        Dim installationElement = xml.<vsix:PackageManifest>.<vsix:Installation>.Single()
+
+        ' We want to modify the .vsixmanifest to say this was installed via MSI
+        installationElement.@InstalledByMsi = "true"
+
+        ' Ensure the VSIX isn't shown in the extension gallery
+        installationElement.@SystemComponent = "true"
+
+        ' We build our VSIXes with the experimental flag so you can install them as test extensions. In the real MSI, they're not experimental.
+        installationElement.Attribute("Experimental")?.Remove()
+
+        ' Update the path to our MEF/Analyzer Components to be in their new home under PrivateAssemblies
+        Dim assets = From asset In xml...<vsix:Asset>
```

Comment:
that it is a rollback slightly alleviates my fear of what this is doing, but still :cry: